image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -21,7 +22,7 @@ object FileMetadataReader {
   private implicit val ctx: ExecutionContext =
     ExecutionContext.fromExecutor(Executors.newCachedThreadPool)
 
-  def fromIPTCHeaders(image: File): Future[FileMetadata] =
+  def fromIPTCHeaders(image: File, mimeType: Option[String]): Future[FileMetadata] =
```

Comment:
Is this data were trying to extract in the IPTC headers? If not this method is misnamed, or extracting the PNG data needs to go in another method.